x/distribution/keeper/abci.go

```diff
@@ -0,0 +1,43 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/distribution/types"
+)
+
+const truncationBlockInterval = 1000
```

Comment:
This is what was defined before (not as a constant).  I am not sure why this number was chosen - I'm assuming arbitrary